### ISSUE TITLE
Forum wide view toggle

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#4193]
For [wide tables](https://pioneerapp.xyz/#/forum/thread/68?post=754) it can be useful to be able to enable to a wide view.

Currently the table is scrolled horizontally, even on wide screens. It's only complete on 50% zoom but then font is very small.

![widetable](https://user-images.githubusercontent.com/31551045/220117416-d6c357b4-03d5-4dc8-8c8c-150064c063a4.png)

**Comment by dmtrjsg** [comment]
Im not sure if that needs to be handled via post width feature, even github doesn't do it. 

How about browser zoom to 50 and zoom in the screen viewport to fit it in: 
<img width="1401" alt="Screenshot 2023-02-24 at 16 50 36" src="https://user-images.githubusercontent.com/93924525/221183169-78ee03ec-cef3-4520-9f04-fd09274d6d32.png">

**Comment by traumschule** [comment]
> How about browser zoom to 50 and zoom in the screen viewport to fit it in:

Yes, it's a temporary hack i use as well but it worsens the need to increase font size (#4094).
There are smart UX ways to give users control over design. I prefer this open to brainstorm solutions for #1794.

**Comment by traumschule** [comment]
Some apps offer a fullscreen mode for tables and graphs. Possibly worth to think about wrt #1794.

**Comment by dmtrjsg** [comment]
Well I added a higher priority suggestion on the font size, as its really low effort high impact. But I'm not sure how this can be handled at this point. 

We could ofc add a button to simply remove the margin. Let our designers think about it and suggest a few ways to handle it and we can triangulate this having design in mind. 

@traumschule do you have an example of web apps offering this feature?

**Comment by traumschule** [comment]
On notion you can toggle a table to be wider.
